--- origsrc/gnome-split-1.2/configure	2013-08-02 12:06:47.663228700 -0500
+++ src/gnome-split-1.2/configure	2013-08-02 12:06:27.887097600 -0500
@@ -140,6 +140,8 @@ sub check_prereq(\@$$@) {
 			$str .= "pkgadd";
 		} elsif ($os eq "arch") {
 			$str .= "pacman -S"
+		} elsif ($os eq "cygwin") {
+			$str .= "setup.exe -P"
 		} else {
 			$str .= "[FIXME fetch and install command for this OS]"; 
 		}
@@ -462,6 +464,9 @@ if ((-f "/etc/gentoo-release") || (-f "/
 } elsif (-f "/etc/redhat-release") {
     output "RedHat";
     $os = "fedora";
+} elsif (-f "/usr/bin/cygwin1.dll") {
+    output "Cygwin";
+    $os = "cygwin";
 } elsif (-f "/etc/java/jpackage-release") {
     output "JPackage";
     $os = "fedora";
@@ -532,6 +537,11 @@ if ($os eq "gentoo") {
 		"java-gnome library",
 		"libjava-gnome-java",
 		"/usr/share/java/gtk.jar");
+} elsif ($os eq "cygwin") {
+	check_prereq(@java_gnome_jar,
+		"java-gnome library",
+		"java-gnome4.1",
+		"/usr/share/java/gtk-4.1.jar");
 } else {
 	bail "failed!", "This OS not configured with defaults!\nTHIS IS AN INTERNAL ERROR, PLEASE FILE A BUG.";
 }
@@ -824,6 +834,15 @@ if ($os eq "gentoo") {
 
 	check_jar($jar, "$vendor jar", $jar_candidate, "");
 	check_javadoc($javadoc, "$vendor javadoc", $javadoc_candidate, "");
+} elsif ($os eq "cygwin") {
+	# check ecj, the standalone Eclipse compiler.
+#	check_compiler($ecj, "Eclipse ecj", which("ecj"), "-g -preserveAllLocals -nowarn -source 1.5 -target 1.5");
+
+	# check javac, the OpenJDK compiler (from jdk6-langtools)
+	check_compiler($javac, "OpenJDK javac", "/usr/bin/javac", "-g");
+
+	check_jar($jar, "Fastjar", "/usr/bin/fastjar", "");
+	check_javadoc($javadoc, "OpenJDK javadoc", "/usr/bin/javadoc", "");
 } else {
 	bail "failed!", "This OS not configured with a workable Java compiler checks!\nTHIS IS AN INTERNAL ERROR, PLEASE FILE A BUG.";
 }
@@ -1099,6 +1118,8 @@ if ($os eq "gentoo") {
 		$cacao_candidate = "/usr/bin/cacao";
 	}
 	check_runtime($cacao, "CACAO VM", $cacao_candidate, "");
+} elsif ($os eq "cygwin") {
+	check_runtime($jamvm, "JamVM VM", "/usr/bin/jamvm", "");
 } else {
 	bail "failed!", "This OS not configured with appropriate Java VM checks!\nTHIS IS AN INTERNAL ERROR, PLEASE FILE A BUG.";
 }
